HousingSearchApi/V1/Boundary/Requests/Validation/GetAccountListRequestValidator.cs

```diff
@@ -7,10 +7,6 @@ public class GetAccountListRequestValidator : AbstractValidator<GetAccountListRe
     {
         public GetAccountListRequestValidator()
         {
-            RuleFor(x => x.SearchText).NotNull()
```

Comment:
Did you mean to remove this?

Reply:
because it depends to the target id in the GetAccountSearchRequest and i handled it inside the Query Generator class

Reply:
is it not equally a validation rule?

Reply:
not needed here 
it has been implemented in the querygenerator class
because depends to the targetId (the same as searchText)